entrypoint.sh

```diff
@@ -10,4 +10,10 @@ then
 
   git commit -m "Auto commit by rick's bot, ci skip"
   git push https://${2}:${GH_TOKEN}@github.com/${3}/${4}.git HEAD:master
+  git commit -m "Auto commit by rick's bot, ci skip"
+  if [ ! -n "$10" ] ;then
+    git push https://${2}:${GH_TOKEN}@github.com/${3}/${4}.git HEAD:master
+  else
+    git push https://${2}:${GH_TOKEN}@github.com/${3}/${4}.git HEAD:"$10"
+  fi
```

Comment:
Actually, the branch name is not necessary. We can just delete it. I have tested it.

See also:
* https://github.com/linuxsuren-bot/test/blob/main/.github/workflows/build.yaml
* https://github.com/linuxsuren-bot/yaml-readme/blob/linuxsuren-bot-patch-1/entrypoint.sh